src/escape_sequences/multibyte_dispatch.c

```diff
@@ -0,0 +1,32 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   multibyte_dispatch.c                               :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: asyed <asyed@student.42.fr>                +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2018/03/25 16:26:22 by nkouris           #+#    #+#             */
+/*   Updated: 2018/04/09 19:32:15 by nkouris          ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "ft_escape.h"
+
+inline __attribute__((always_inline)) int	multibyte_dispatch(char byte)
+{
+	if (byte == 67 || byte == 68 || byte == 70 || byte == 72)
```

Comment:
Uncommented numbers are confusing and make code hard to read. Either comment what the codes are or make Macros for each.

Reply:
I'll throw a comment in that they correspond to the ascii table and switch to hexadecimal representations